crates/cli/src/subcommands/login.rs

```diff
@@ -7,16 +7,22 @@ use webbrowser;
 pub fn cli() -> Command {
     Command::new("login")
         .arg(
-            Arg::new("host")
-                .long("host")
+            Arg::new("auth-host")
+                .long("auth-host")
                 .default_value("https://spacetimedb.com")
                 .help("Fetch login token from a different host"),
         )
+        .arg(
+            Arg::new("server")
+                .long("direct")
```

Comment:
Is "server" a typo? Why is "direct" the long of "server". Also I think perhaps something like "cluster-issued-login" or "server-issued-login" is a more apt name.

Reply:
In the CLI helptext, the arg name is used as the name of the value, whereas the `long` is the name of the param itself:

```
Options:
      --auth-host <auth-host>      Fetch login token from a different host [default: https://spacetimedb.com]
      --direct <server>            Log in to a SpacetimeDB server directly, without going through a global auth server
      --token <spacetimedb-token>  Bypass the login flow and use a login token directly
      --refresh-cache              Clear the cached tokens and re-fetch them
  -h, --help                       Print help
```

I'm happy with a more descriptive name!

Reply:
(renamed to `--server-issued-login`)